### PR TITLE
added cors and prettier

### PR DESCRIPTION
**Pull request by Zain-ul-din**
Cors: https://www.npmjs.com/package/cors
Prettier is a dev dependency package that helps you to indent code

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 
 const app = express();
 
-app.use(GCRouter);
-app.use(TimetableRouter);
-app.use(PopulateRouter);
-
-app.listen(3000, () => {
+app
+.use (cors ({})) // allow all
```
Comment (IIvexII) [inline src/index.js:9]: Are you sure it will work? Have you tested it before creating pull request?

**Review by IIvexII — COMMENTED** [review]
Please remove the prettier It is unnecessary to have it. Thanks

**Review by IIvexII — APPROVED** [review]
LGTM :tada:

File: src/index.js (hunk 1 above)
Comment (Zain-ul-din) [reply src/index.js:9]: yeah

**Pull request merged**